src/main/java/com/devathon/preguntonic/model/PlayerStatus.java

```diff
@@ -0,0 +1,10 @@
+package com.devathon.preguntonic.model;
+
+public enum PlayerStatus {
```

Comment:
I like this PlayerStatus, but WDYT to also used it in `Lobby` to handle the `is_ready` state and also remove this bolean param.

Reply:
We can add `PlayerStatus.READY` to do that. WDYT

Reply:
I like it 👍